README.md

```diff
@@ -1,3 +1,3 @@
 # ev-checker #
 -----
-Deprecated. Use https://tls-observatory.services.mozilla.com/static/ev-checker.html instead.
+Deprecated. Use https://evready-dot-ccadb-231121.uc.r.appspot.com/evready instead.
```

Comment:
The link on https://wiki.mozilla.org/PSM:EV_Testing_Easy_Version is https://evready-dot-ccadb-231121.appspot.com/evready - should we use that instead?

Reply:
I think so, but I'll defer to others.

Reply:
Linking to the wiki page sounds good.